rules/TypeDeclaration/NodeAnalyzer/CallTypesResolver.php

```diff
@@ -76,7 +76,13 @@ public function resolveTypesFromCalls(array $calls): array
                     continue;
                 }
 
-                $staticTypesByArgumentPosition[$position][] = $this->resolveArgValueType($arg);
+                if ($arg->name instanceof Identifier) {
+                    $positionOrName = $arg->name->toString();
+                } else {
+                    $positionOrName = $position;
+                }
+
+                $staticTypesByArgumentPosition[$positionOrName][] = $this->resolveArgValueType($arg);
```

Comment:
There is `ArgsAnalyzer->resolveArgPosition()`

https://github.com/rectorphp/rector-src/blob/966cfd99a205b464549705b8a61f5698cc22bb78/src/NodeAnalyzer/ArgsAnalyzer.php#L35

I will create a PR for it.